.github/workflows/windows_build_with_gtest.yml

```diff
@@ -3,14 +3,6 @@ name: win_gtest
 on:
   # allow manually trigger
   workflow_dispatch:
-  push:
-    branches:
-      - main
-      - 'maint-**'
-  pull_request:
-    branches:
-      - main
-      - 'maint-**'
```

Comment:
no triggers at all ?

Reply:
Only workflo dispatch currently. This is to be discussed. Also this is gtest for baremetal deployment.